android_fonts/vue-virtual-scroll-list.js

```diff
@@ -0,0 +1,602 @@
+// taken from https://github.com/tangbc/vue-virtual-scroll-list/blob/master/dist/index.js
```

Comment:
we don't have to do something about license, put it into a /third_party dir or something?

Reply:
https://github.com/tangbc/vue-virtual-scroll-list/blob/master/LICENSE MIT license.
I think for now it's okay to have it laying around. We'll re-do this page soon enough.

Reply:
Lets at least shove it into a third_party dir and reference the license, similar to https://github.com/googlefonts/noto-emoji/tree/master/third_party

Reply:
Should be fine to do post-merge.